### PR TITLE
Add steps for how to upload plugins in HA

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

**Comment by levb** [comment]
Once merged, please reference in https://mattermost.atlassian.net/browse/MM-8680 to update when we have a better solution.

File: source/administration/plugins.rst [hunk 1]
```diff
@@ -53,20 +53,18 @@ Mattermost supports plugin uploads by System Admins, which allow you to customiz
...
 
 1) Through System Console UI:
  - Log in to Mattermost as a System Admin.
- - Navigate to **Plugins > Management** and upload the `plugin.tar.gz` you generated above.
+ - Navigate to **Plugins > Management** and upload ``plugin.tar.gz``.
  - Click "Activate" under the plugin after it has uploaded.
 
 2) Manually:
- - Extract `plugin.tar.gz` to a folder with the same name as the plugin id you specified in ``plugin.json/plugin.json``.
+ - Extract ``plugin.tar.gz`` to a folder with the same name as the plugin id you specified in ``plugin.json/plugin.json``.
```
Comment (icelander) [inline source/administration/plugins.rst]: Where is the `plugin.json/plugin.json` file located?

Comment (jasonblais) [reply source/administration/plugins.rst]: Updated, let me know if that helps clarify its location and overall directory structure.

Comment (amyblais) [reply source/administration/plugins.rst]: @icelander Do the updates look ok?

**Pull request merged**